app/views/form/_check_answers_summary_list.html.erb

```diff
@@ -3,7 +3,12 @@
     <% summary_list.row do |row| %>
       <% row.key { question.check_answer_label.to_s.presence || question.header.to_s } %>
       <% row.value do %>
-        <%= get_answer_label(question, @case_log) %><br>
+        <%= get_answer_label(question, @case_log) %>
```

Comment:
This should match what we do in property information, with a larger space between the value and the inferred value – should be something like this:

```suggestion
        <span class="govuk-!-margin-right-4"><%= get_answer_label(question, @case_log) %></span>
```